CODE_OF_CONDUCT.md

```diff
@@ -1,36 +1,114 @@
 # Contributor Covenant Code of Conduct
 
 ## Our Pledge
+Diversity is one of the greatest strengths a community can have and many times
+that strength is born from the friction that can only come through sharing of
+differing perspectives.
 
-In the interest of fostering an open and welcoming environment, we as
+In the interest of fostering an open, welcoming, and encouraging environment, we as
 contributors and maintainers pledge to making participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
 size, disability, ethnicity, gender identity and expression, level of experience,
 nationality, personal appearance, race, religion, or sexual identity and
 orientation.
 
 ## Our Standards
+Habitat is a diverse community of professionals and volunteers who come from all
+over the world to make Habitat better. Community members may fulfill many roles
+including mentoring, teaching, and connecting with other members of the community.
 
-Examples of behavior that contributes to creating a positive environment
-include:
+Be careful in the words that you choose. Be kind to others. Practice empathy. Don't
+insult or put down others. Remember that sexist, racist, and other exclusionary jokes
+can be offensive to those around you. If you think your conversation is making another
+community member uncomfortable _or_ if they tell you so, stop immediately, make amends,
+and move forward.
 
+As you are working with other members of the community, please keep in mind the following guidelines, which apply equally to founders, mentors, those who submit new features and pull requests, and to anyone who is seeking help and guidance.
+
+The following list isn’t exhaustive; but these few examples can help all of us communicate well so that the community can work better together:
 * Using welcoming and inclusive language
+* Exercising patience and friendliness
 * Being respectful of differing viewpoints and experiences
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members
 
-Examples of unacceptable behavior by participants include:
+The previous list applies to all forms of communication: Slack (or any web chat), discourse, the issue tracker, and any other forum that is used by the community.
+
+Please keep in mind that:
+* Your work will be used by other people, and you, in turn, will depend on the work of others.
+* Decisions that you make will often affect others in the community.
+* Disagreements happen, but should not be an excuse for poor behavior and bad manners. When disagreements do happen, let’s work together to solve them effectively and in a way that ensures that everyone understands what the disagreement was.
+* People may not understand jokes, sarcasm, and oblique references in the same way that you do. So remember that and be kind to the other members of the community.
```

Comment:
I'll agree with the sentiment of the tweet, need to rephrase it.

How about a bit of paraphrasing:

> * Our community spans languages, cultures, perspectives and many other facets, and sometimes inside jokes, sarcastic comments, or oblique references may not be perceived as planned by the author. Keep this in mind, and be kind to others, treat them with the same respect you would expect to receive.